Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -261,23 +261,67 @@ NS_SWIFT_NAME(Firestore)
 
 #pragma mark - Bundles
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param bundleData Data from the bundle to be loaded.
+ * @return A `FIRLoadBundleTask` (`LoadBundleTask` in Swift), which allow registering observers
```

Comment:
Nits:
* s/allow/allows/;
* I think the comma before `which` is unnecessary;
* s/registering/registered/;
* I think the comma after `updates` is unnecessary.